CBLClient/RestClient.py

```diff
@@ -0,0 +1,43 @@
+from requests import Session
```

Comment:
Why is this file needed? The APIs here are already part of other .py files

Reply:
deleted